face_detector/src/face_detection.cpp

```diff
@@ -272,7 +277,7 @@ class FaceDetector
       faces_->initFaceDetectionDisparity(1, haar_filename_, face_size_min_m, face_size_max_m, max_face_z_m, face_sep_dist_m);
 
       stereo_namespace_ = nh_.resolveName("camera");
-      image_image_ = nh_.resolveName("image");
+      image_image_ = nh_.resolveName("image_topic");
```

Comment:
I have no idea why in this line the parameter that refers to the name of an image topic for stereo was "image", different from the one used for RGBD ("image_topic". See https://github.com/wg-perception/people/pull/30/files#diff-837a1f218ba855450bffac72191509f7R243). So with this change we only use "image_topic".